docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -105,9 +105,14 @@ Findable
 		
 	- Authorship, contact information, and description:
 	
-		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through. 
+		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through.
+		
+	- Link GitHub repository:
+	
+		- Additionally, for workflow languages that include meta sections, you can include a URL to your original GitHub repo README in the meta section of your descriptor file(s). In case of multiple descriptor files, use the primary descriptor file to host this information. If your workflow is moved out of Dockstore, this will help connect it with the original source code. You may consider doing so especially when you have additional comprehensive README files available on GitHub.
```

Comment:
`If your workflow is moved out of Dockstore, this will help connect it with the original source code. You may consider doing so especially when you have additional comprehensive README files available on GitHub.`

Still not sure if I follow this. 
Perhaps this means if the workflow is copied (as opposed to moved out of)  from Dockstore, say to be launched. URLs in the descriptor could be helpful to indicate where it came from.

Reply:
I can change the wording to make it more clear. But like you said, the workflow could be "copied" or "downloaded" from Dockstore to run locally or on an HPC maybe.

What do you think about:

"If your workflow is downloaded or copied from Dockstore to be run on a different computing environment such as on a local machine or HPC, this will help connect it with the original source code.

Reply:
Yeah, I think it is just the word "moved" which implies something different to me.